pkg/azurefile/azure.go

```diff
@@ -49,6 +50,23 @@ var (
 	storageService = "Microsoft.Storage"
 )
 
+func getRuntimeClassForPod(ctx context.Context, kubeClient clientset.Interface, podName string, podNameSpace string) (string, error) {
+	if kubeClient == nil {
+		klog.Warningf("kubeClient is nil")
```

Comment:
not necessary to log every warning or error msg in util function since it's already logged in outside caller